src/guide/tutorials/rust.md

```diff
@@ -58,7 +58,7 @@ first.
 ::: info
 
 You could also make use of our `test_network` crate, which is available via
-[GitHub](https://github.com/hyperledger/iroha/tree/main/core/test_network)
+[GitHub](https://github.com/hyperledger-iroha/iroha/tree/main/core/test_network)
```

Comment:
Double check the branch, this path does not exist on the `main` branch.

Reply:
Perhaps, the `test_network` crate should be added to the `main` branch?
@nxsaken, what do you think?

Reply:
The path has been changed to https://github.com/hyperledger-iroha/iroha/tree/main/crates/iroha_test_network

Reply:
```suggestion
[GitHub](https://github.com/hyperledger-iroha/iroha/tree/main/crates/iroha_test_network)
```